lib/commands/context.js

```diff
@@ -7,8 +7,11 @@ import { util } from 'appium-support';
 import { errors } from 'appium-base-driver';
 import {
   default as webviewHelpers,
-  NATIVE_WIN, WEBVIEW_BASE, WEBVIEW_WIN, CHROMIUM_WIN
+  NATIVE_WIN, WEBVIEW_BASE, WEBVIEW_WIN, CHROMIUM_WIN, KNOWN_CHROME_PACKAGE_NAMES
 } from '../webview-helpers';
+import {
+  APP_STATE_RUNNING_IN_BACKGROUND, APP_STATE_RUNNING_IN_FOREGROUND
```

Comment:
consider moving these constant into a separate file. command modules should not import each other

Reply:
Thanks for your advice. Addressed as https://github.com/appium/appium-android-driver/pull/699/commits/2b80397adca17407ecabe3aca0c9765e19bca940